cmd/controller-manager/app/controller_manager.go

```diff
@@ -248,7 +248,7 @@ func Run(controllerManagerOptions *options.ControllerManagerServer) error {
 // groups are available in the endpoint reachable from the given client and
 // returns a map of them.
 func getAvailableResources(clientBuilder controller.ClientBuilder) (map[schema.GroupVersionResource]bool, error) {
-	var discoveryClient discovery.DiscoveryInterface
+	var resourceMap []*metav1.APIResourceList
```

Comment:
Do we still need this var? Doesn't look like we do anything with it.

Reply:
yes, line 278